platform/core/src/services/UIDialogService/index.js

```diff
@@ -54,6 +56,8 @@ function create({
   onStart,
   onDrag,
   onStop,
+  centralize = false,
+  useLastPosition = true,
```

Comment:
Are there instances where `isDraggable` would be true, but this would be `false`?
Maybe instead of having the service remember the dialog's last position, we should have it return it's position when it closes? Then the caller can track and use it as initial values when opening if it wants this behavior?

Reply:
by this you mean lastPosition or centralize? isDraggable is independent of them, it just freezes  the component in a default position or last position if required.  I was wondering if the last position of a dialog would be better living outside... because this would require a new state in every component that needs this behaviour.